tools/synch.py

```diff
@@ -46,60 +46,8 @@
 # Code that does have a mirror in the mbed SDK
 # Tuple data: (repo_name, list_of_code_dirs, [team])
 # team is optional - if not specified, the code is published under mbed_official
-OFFICIAL_CODE = (
-    ("mbed-dev" , [MBED_DRIVERS, MBED_PLATFORM, MBED_HAL]),
-    ("mbed-rtos", RTOS),
```

Comment:
arggh neither of the latter two commits should have come in, they came as a result of playing with something else! Will remove and re-do this PR !!No we no longer synchronise mbed-rtos .